portfolio/src/main/webapp/script.js

```diff
@@ -92,24 +93,76 @@ function stop() {
 }
 
 /*
- * Fetches a greeting from the server and passes it to handleResponse().
+ * Fetches comments from the server and adds them to the DOM.
  */
-function getGreeting() {
-    const responsePromise = fetch('/data');
-    responsePromise.then(handleReponse)
+function loadComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    const commentContainer = document.getElementById('comment-container');
+    comments.forEach((comment) => {
+      commentContainer.appendChild(createComment(comment));
```

Comment:
Nice. Looping over the comments is the correct approach here, and as I understand, not the one recommended by the starter guide :)